backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardMutationService.kt

```diff
@@ -74,25 +77,25 @@ class CardMutationService {
         if (!cardInfo.extensions.hasExtensionRegion()) {
             throw InvalidCardHashException()
         }
-        return transaction {
```

Comment:
moved the transaction up a level to keep all actions in a single transaction:
- revoke old cards
- create dynamic card
- create static card